schnapps/core/view.cpp

```diff
@@ -385,6 +396,8 @@ void View::preDraw()
 
 void View::draw()
 {
+	glClearColor(background_color_.redF(), background_color_.greenF(), background_color_.blueF(), background_color_.alphaF());
```

Comment:
Plutôt que d'appeler `glClearColor` dans chaque `draw`, est-ce qu'on ne peut pas l'appeler uniquement dans la méthode `color_selected` c'est-à-dire seulement quand la couleur du background change ? (il faudra peut-être faire un `this->makeCurrent()` avant).